src/vs/workbench/contrib/search/browser/searchActionsFind.ts

```diff
@@ -381,11 +381,11 @@ export async function findInFilesCommand(accessor: ServicesAccessor, _args: IFin
 				const searchAndReplaceWidget = openedView.searchAndReplaceWidget;
 				searchAndReplaceWidget.toggleReplace(typeof args.replace === 'string');
 				let updatedText = false;
-				if (typeof args.query === 'string') {
-					openedView.setSearchParameters(args);
-				} else {
+				if (typeof args.query !== 'string') {
```

Comment:
FYI we have a util for this, using it should end up being less bytes in the bundle

https://github.com/microsoft/vscode/blob/7b39d0e24ae473beb9d1d28d8372e960681ddda9/src/vs/base/common/types.ts#L9-L11